src/ploomber/tasks/notebook.py

```diff
@@ -268,9 +271,28 @@ def _from_ipynb(path_to_nb, exporter, nbconvert_export_kwargs):
 
 
 class NotebookMixin(FileLoaderMixin):
+
+    @staticmethod
+    def _validate_parameters_cell(source,
+                                  extract_up=False,
+                                  extract_prod=False):
```

Comment:
please document extract_up and extract_prod, also rename them for clarity to extract_upstream and extract_product

description: flags used to determine the content of the parameters cell, only used if the notebook is missing the parameters cell